hooks/pre_validations/34-pulpcore_directory_layout.rb

```diff
@@ -0,0 +1,9 @@
+require 'pathname'
+
+PULP_ROOT = Pathname.new('/var/lib/pulp')
+LEGACY_DIR = PULP_ROOT / 'artifact'
+DESTINATION = PULP_ROOT / 'media' / LEGACY_DIR.basename
+
+if LEGACY_DIR.directory? && !LEGACY_DIR.symlink? && DESTINATION.directory?
+  fail_and_exit("Target #{DESTINATION} already exists. Unable to migrate Pulp media root.")
```

Comment:
What action should a user take when getting this message?

Reply:
My last remaining question before merging.

Reply:
How about now?